src/field_theory/minimal_polynomial.lean

```diff
@@ -274,6 +278,51 @@ begin
   { exact monic_map _ (monic hx) }
 end
 
+/-- The minimal polynomial over `ℤ` is the same as the minimal polynomial over `ℚ`. -/
+--TODO use `gcd_domain_eq_field_fractions` directly
```

Comment:
I tried doing so and it wasn't easy, since we need `fraction_map.int.fraction_map.codomain` to unify with ℚ a lot. There has been an idea on the Zulip chat floating around, to make `localization_map` extend `algebra` instead of `ring_hom`, which should make all this codomain business obsolete. So let's keep the todo, but indicate when it's possible to fix it.

```suggestion
--TODO use `gcd_domain_eq_field_fractions` directly when localizations are defined
-- in terms of algebras instead of `ring_hom`s
```